### PR TITLE
Update Presto documentation link

### PR DESCRIPTION
**Pull request by djsstarburst**
Current and relatively complete documentation for the Iceberg Presto
connector was added to Presto in July.  This PR removes the pointer
to the obsolete Presto README.md file, and changes the link to point
to the Iceberg Presto connector documentation.

### CODE REVIEW

File: site/docs/presto.md [hunk 1]
```diff
@@ -17,4 +17,4 @@
 
 # Presto
 
-Iceberg connector is part of Presto SQL's master branch, although support for many features is still under development. Current status of the connector is detailed in the [presto-iceberg README](https://github.com/prestosql/presto/blob/master/presto-iceberg/README.md).
+Iceberg connector is part of Presto SQL's master branch, and feature development is complete.  Connector documentation can be found in [Presto Iceberg Connector](https://github.com/prestosql/presto/blob/master/presto-docs/src/main/sphinx/connector/iceberg.rst).
```
Comment (electrum) [inline site/docs/presto.md:20]: We should link to the published documentation and not the source code: https://prestosql.io/docs/current/connector/iceberg.html

Comment (djsstarburst) [reply site/docs/presto.md]: Good point; changed and force-pushed.

File: site/docs/presto.md [hunk 2]
```diff
@@ -17,4 +17,4 @@
 
 # Presto
 
-Iceberg connector is part of Presto SQL's master branch, although support for many features is still under development. Current status of the connector is detailed in the [presto-iceberg README](https://github.com/prestosql/presto/blob/master/presto-iceberg/README.md).
+Iceberg connector is part of Presto SQL's master branch, and feature development is complete.  Connector documentation can be found in [Presto Iceberg Connector](https://prestosql.io/docs/current/connector/iceberg.html).
```
Comment (mosabua) [inline site/docs/presto.md:20]: How about rewording it a bit to something like this:

```
The Iceberg connector is available with Presto SQL. It is feature complete and usage details can be found in the [documentation](https://prestosql.io/docs/current/connector/iceberg.html).
```

Comment (djsstarburst) [reply site/docs/presto.md]: Indeed that's better; updated and force-pushed.

**Comment by rdblue** [comment]
Thanks, @djsstarburst! Great to have this updated.

Are there any plans to add documentation for Presto to this page? If not, then we might reformat this to just link from the top bar rather than having a page with the link.

**Comment by electrum** [comment]
@rdblue good idea. I agree it’s better to just link to the Presto documentation, since it’s complete and always up to date.
